src/SDKs/DeploymentManager/DeploymentManager.Tests/Management.DeploymentManager.Tests.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DeploymentManager.Tests</PackageId>
+    <Description>DeploymentManager.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.DeploymentManager.Tests</AssemblyName>
+    <Version>0.9.0-preview</Version>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <ProjectGuid>{23CD2878-1A27-4ECD-80F9-8B9BB1539F7F}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+  
+  <ItemGroup>       
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="4.2.0-preview" />
+    <PackageReference Include="WindowsAzure.Storage" Version="8.7.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Authorization\Management.Authorization\Microsoft.Azure.Management.Authorization.csproj" />
```

Comment:
Why is this dependency required. Also, reference a nuget package instead of a different project like so.

Reply:
This dependency is required to have scenario tests for certain resources. Same for all such dependencies introduced.

Reply:
If and when these RPs release their next versions they might break you. Please reference their published Nuget packages in order to avoid it.